edge-patron/src/main/resources/karate-config.js

```diff
@@ -0,0 +1,68 @@
+function fn() {
+
+  karate.configure('logPrettyRequest', true);
+  var retryConfig = { count: 20, interval: 30000 }
+  karate.configure('retry', retryConfig)
+
+  var env = karate.env;
+
+  // The "testTenant" property could be specified during test runs
+  var testTenant = karate.properties['testTenant'];
+
+  var config = {
+    baseUrl: 'http://localhost:9130',
+    admin: {tenant: 'diku', name: 'diku_admin', password: 'admin'},
+
+    testTenant: testTenant ? testTenant: 'testTenant',
+    testAdmin: {tenant: testTenant, name: 'test-admin', password: 'admin'},
+    testUser: {tenant: testTenant, name: 'test-user', password: 'test'},
+
+    // define global features
+    login: karate.read('classpath:common/login.feature'),
+    dev: karate.read('classpath:common/dev.feature'),
+
+    // define global functions
+    random_string: function() {
+      var text = "";
+      var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+      for (var i = 0; i < 8; i++)
+        text += possible.charAt(Math.floor(Math.random() * possible.length));
+      return text;
+    },
+    //to generate random barcode
+    random_numbers: function() {
+      return Math.floor(Math.random() * 1000000);
+    },
+    random_uuid: function() {
+      return java.util.UUID.randomUUID() + '';
+    }
+  };
+  if (env == 'testing') {
+    config.baseUrl = 'https://folio-testing-okapi.dev.folio.org:443';
+    config.edgeUrl = 'https://folio-testing.dev.folio.org:8000';
+    config.apikey = 'eyJzIjoiNXNlNGdnbXk1TiIsInQiOiJkaWt1IiwidSI6ImRpa3UifQ==';
+    config.admin = {
+      tenant: 'diku',
+      name: 'diku_admin',
+      password: 'admin'
+    }
```

Comment:
I thought these tests created a new tenant each time they ran. Why is the diku tenant involved in these tests?